### DISCUSSION TITLE
LOGB=logbase,offset

### DISCUSSION THREAD

**Discussion by cedricchangy** [gopro/labs#215] [General]
Hi, I'm trying to use the experimental log, whatever the settings I use there is no difference.
Thanks for the help

**Comment by dnewman-gpsw** [comment]
What command are are trying, on what camera, and firmware version?

**Reply by cedricchangy** [reply]
thanks for the reply ! 
I try with a gopro herro 10 ith last last labs firmare and I try few command :  !MLOGB=1000 !MLOGB=400

**Reply by dnewman-gpsw** [reply]
LOGB is for the HERO11 only, as stated on https://gopro.github.io/labs/control/extensions/ under **"HERO11 and Mini cameras - Extensions in the latest Labs firmware (Nov ‘22)"**   The issue with an enhanced log curve, it really requires 10-bit, otherwise it can introduce more banding in the highlights.  

The release notes for each Labs camera release are here https://gopro.github.io/labs/control/notes/
